packages/backend-core/src/tenancy/tenancy.ts

```diff
@@ -93,11 +93,19 @@ export const getTenantIDFromCtx = (
   // subdomain
   if (isAllowed(TenantResolutionStrategy.SUBDOMAIN)) {
     // e.g. budibase.app or local.com:10000
-    const platformHost = new URL(getPlatformURL()).host.split(":")[0]
+    let platformHost
+    try {
+      platformHost = new URL(getPlatformURL()).host.split(":")[0]
+    } catch (err: any) {
+      // if invalid URL, just don't try to process subdomain
+      if (err.code !== "ERR_INVALID_URL") {
```

Comment:
In which case we want to support invalid URLs?

Reply:
Just local development, it won't happen in multi-tenant environments, but this was blocking the ability to fetch tenant IDs in other ways